### PR TITLE
CS: fixes

### PR DESCRIPTION
**Pull request by keradus**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | ? |
| Fixed tickets | N/A |
| License | MIT |
| Doc PR | N/A |

Update after releasing PHP CS Fixer 1.6.

To keep fabbot.io happy ;)

### CODE REVIEW

File: src/Symfony/Component/Finder/Tests/Iterator/FilePathsIteratorTest.php [hunk 1]
```diff
@@ -44,7 +44,7 @@ public function getSubPathData()
                     $tmpDir.DIRECTORY_SEPARATOR.'test.php' => $tmpDir.DIRECTORY_SEPARATOR.'test.php',
                     $tmpDir.DIRECTORY_SEPARATOR.'toto' => $tmpDir.DIRECTORY_SEPARATOR.'toto',
                 ),
-                array( // subPaths
+                array(// subPaths
```
Comment (fabpot) [inline src/Symfony/Component/Finder/Tests/Iterator/FilePathsIteratorTest.php]: I kind of like the space here. Another fix could be to put the comment on another line:

```php
array(
    // subPaths
    ...
```

Comment (keradus) [inline src/Symfony/Component/Finder/Tests/Iterator/FilePathsIteratorTest.php]: done

**Comment by fabpot** [comment]
Thank you @keradus.

**Comment by keradus** [comment]
Any chances to merge 2.3 into 2.6?  ;) that would be great!

**Comment by fabpot** [comment]
Done

**Comment by keradus** [comment]
Thanks ;)

**Pull request merged**